src/providers/settingsProvider.js

```diff
@@ -30,8 +31,45 @@ function setInitialValue(settingName, initialValue) {
 }
 
 function onDidChange(key, callback) {
-    store.onDidChange(key, (newValue, oldValue) => {
-        callback({ newValue: newValue, oldValue: oldValue })
+    return store.onDidChange(key, (newValue, oldValue) => {
+        callback({ newValue, oldValue })
```

Comment:
![image](https://user-images.githubusercontent.com/2112638/91479862-7bdcaa00-e878-11ea-9010-df89bb1b0a2d.png)

changing to callback(newValue, oldValue)

![image](https://user-images.githubusercontent.com/2112638/91479958-a29ae080-e878-11ea-9ed4-fe5266403e65.png)

Reply:
Fixed 😄